test/src/module/storage/azureTest.c

```diff
@@ -214,7 +214,7 @@ testRun(void)
             (StorageAzure *)storageDriver(
                 storageAzureNew(
                     STRDEF("/repo"), false, NULL, TEST_CONTAINER_STR, TEST_ACCOUNT_STR, storageAzureKeyTypeShared,
-                    TEST_KEY_SHARED_STR, 16, NULL, 443, 1000, true, NULL, NULL)),
+                    TEST_KEY_SHARED_STR, 16, NULL, STRDEF("blob.core.windows.net"), 443, 1000, true, NULL, NULL)),
```

Comment:
Here, I would like to see us use a default as stated in my previous comment - e.g. CFGOPTVAL_REPO_ENDPOINT_DEFAULT or CFGOPTVAL_DEFAULT_REPO_ENDPOINT or something.

Reply:
In these tests the endpoint could be any value since they are just checking to be sure the value gets passed through. I don't think it's worth creating constants for tests like this.